pkg/vsphere/toolbox/vix_command.go

```diff
@@ -23,13 +23,22 @@ import (
 	"fmt"
 	"os"
 	"runtime"
+
+	"github.com/vmware/vic/pkg/vsphere/toolbox/hgfs"
 )
 
 const (
 	vixCommandMagicWord = 0xd00d0001
 
 	vixCommandGetToolsState = 62
-	vixCommandStartProgram  = 185
+
+	vixCommandKillProcess     = 85
```

Comment:
I grabbed this constant along with vixCommandListProcessesEx while I was in there, with doing a proper async vixCommandStartProgram impl.  But the pseudo "kill" command will probably stay as-is, since the TerminateProcessInGuest method (which maps to vixCommandKillProcess), does not take any arguments other than a PID.